vendor.conf

```diff
@@ -1,5 +1,5 @@
 github.com/urfave/cli v1.17.0
-github.com/containers/image master
+github.com/containers/image helpers https://github.com/dlorenc/image
```

Comment:
Is this syntax correct?  What is the extra helpers?

Reply:
`helpers` is the branch in `dlorenc`'s own fork where to fetch the dependency from. This is here till containers/image related PR is merged, afterwards this will be changed back to "master" and revendored correctly.
This process ensures c/image PRs don't break skopeo :)

Reply:
I have never seen this syntax before.  I did not know it was supported.

Reply:
https://github.com/LK4D4/vndr#vendorconf

Reply:
@dlorenc please change this to the original, re-run vndr, push and we should be to go here as well

Reply:
Done. Thanks!

Reply:
@runcom Thanks for the link, now I understand.